### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mlissner**
This cleans up a few formatting issues. Nothing that could cause a bug, but lots of stuff that a PEP8 analysis wasn't pleased about. Should apply cleanly after the previous pull is done. 

Tests continue to pass with these changes.

### CODE REVIEW

File: pdfquery/pdfquery.py [hunk 1]
```diff
@@ -25,36 +25,51 @@
...
+    return all([
+        float(el.get('x0')) <= float(child.get('x0')),
+        float(el.get('x1')) >= float(child.get('x1')),
+        float(el.get('y0')) <= float(child.get('y0')),
+        float(el.get('y1')) >= float(child.get('y1')),
+    ])
 
-_comp_bbox_keys_required = set(['x0', 'x1', 'y0', 'y1'])
 
+_comp_bbox_keys_required = {'x0', 'x1', 'y0', 'y1'}
```
Comment (jcushman) [inline pdfquery/pdfquery.py]: This is done the old-fashioned way to maintain Python 2.6 compatibility (bug #16).

File: pdfquery/pdfquery.py [hunk 2]
```diff
@@ -73,24 +88,32 @@ def _flatten(l, ltypes=(list, tuple)):
...
-    unicode(codecs.BOM_UTF8,'utf8'),
+bom_headers = {
+    unicode(codecs.BOM_UTF8, 'utf8'),
     unicode(codecs.BOM_UTF16_LE, 'utf-16LE'),
     unicode(codecs.BOM_UTF16_BE, 'utf-16BE'),
     unicode(codecs.BOM_UTF32_LE, 'utf-32LE'),
     unicode(codecs.BOM_UTF32_BE, 'utf-32BE')
-])
+}
+
```
Comment (jcushman) [inline pdfquery/pdfquery.py]: Ditto Python 2.6 compat.

**Comment by jcushman** [comment]
Thanks! Happy to pull if you can revert the set() thing, then I can kick out a release with the Annot stuff.

**Comment by mlissner** [comment]
OK, all set! Never knew the `{}` syntax was a new thing. Good to know, I suppose.

**Pull request closed without merging**